### DISCUSSION TITLE
Current screen variable & Apps synchronisation

### DISCUSSION THREAD

**Discussion by floredelasteyrie** [Budibase/budibase#4369] [Help] [answered]
Hello everyone, 

New to Budibase and really enjoying the experience so far, great tool to build products quickly and iterate even faster 🙂.

I have a few questions about things I can't seem to do, I don't know if it's me who hasn't found the feature yet or if they just don't exist!

1. I would like to use the name (or another variable) of my screens in a Layout. For example, I have updated the Navigation Layout to look like this:
<img width="1237" alt="Screenshot 2022-02-07 at 11 29 31" src="https://user-images.githubusercontent.com/41300908/152772758-8d436791-b4b7-4c49-a797-950c1f7cada0.png">

Where I wrote "Current Screen Title ?" I would like to be able to use a dynamic variable like {{ CurrentScreen.title }}. Is this possible?

2. The application I am creating is for customers, to allow them to view their data in some dashboards. For security reasons, I need to segregate data, using a new schema for each client. Because of this, I feel that it would be easiest to have one Budibase App per customer, which would be exactly the same for everyone, but would point to different schemas in the database. 
I managed to do this by creating the App entirely for one client, then exporting and importing it for each new client. But then, when I need to update the App, does that mean I would have to update each of them manually as well? 
I would like to have the possibility to synchronise several Apps automatically, I don't know if this is possible ... Or to be able to export / import back to an existing App, keeping the customer data intact ? If you have any ideas on how to do this, I'm very interested!

Thanks in advance and have a nice day, 
Flore

**Comment by mjashanks** [comment, marked as answer]
Hey Flore - welcome!

Unfortunately, it's bad news for both questions
1. No - we cannot do this, although it's a good shout, we should add this to the available bindings.
2. Again, this is not possible at the moment. There is no way to apply a "Diff" from one app to another. I think that there is potential for us to do this in the future, as one feature we intend to build is "Environments" - whereby you can promote changes from one environment to the next. However, That's way in the future - end of this year at least.

**Reply by floredelasteyrie** [reply]
Thank you for your quick answer, even if it's bad news!

By the way, is your roadmap or dev focus available somewhere, so we know what we can expect as new features in the coming weeks/months? :)

**Reply by mjashanks** [reply]
you're welcome! 

Roadmap here https://github.com/Budibase/budibase/projects/10 - this details all the big features.
